Packs/AzureSecurityCenter/Integrations/AzureSecurityCenter_v2/AzureSecurityCenter_v2.yml

```diff
@@ -120,6 +120,12 @@ script:
       name: alert_id
       required: false
       secret: false
+    - default: false
+      description: The subscription ID to use. Can be retrieved from the azure-sc-list-subscriptions command. If not specified, the default subscription ID is used.
+      isArray: false
+      name: subscription_id
+      required: false
+      secret: false
```

Comment:
Is this the only command in which this argument is missing?

Reply:
yes.